composer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spryker-eco/avalara-tax",
+  "description": "AvalaraTax module",
+  "license": "proprietary",
+  "require": {
+    "php": ">=7.3",
```

Comment:
Should `avalara-tax-extension` module be required?

Reply:
It will be added in the next PRs.